modules/virt-enabling-preallocation-for-dv.adoc

```diff
@@ -24,7 +24,11 @@ spec:
   source: <1>
     pvc:
       preallocation: true <2>
+      name: <pvc_name> <3>
+      namespace: <pvc_namespace> <4>
 # ...
 ----
 <1> All CDI source types support preallocation. However, preallocation is ignored for cloning operations.
```

Comment:
`preallocation is ignored for cloning operations`, but the example yaml shows us the cloning from a PVC. 
Maybe this would be a better example
```
spec:
  source:
      http: 
            ....
```

@awels, you know this topic better than anyone else, could you please review this doc as well?

Reply:
I would probably use a registry source, since most of the other examples in the document use it as well. And yes pre-allocation with cloning is a bit muddied, so something else is definitely better.

Reply:
I am not sure what is the suggested change regarding the source.  

Also, should we strike this line?
```suggestion
<1> All CDI source types support preallocation. 
```

Reply:
@awels Any thoughts here? ^^

Reply:
@awels @jpeimer Do you have an example DataVolume CR that uses a registry source that we could use in the docs?